src/null-safety/migration-guide.md

```diff
@@ -62,7 +62,7 @@ The instructions assume your code is under **source control**,
 so that you can easily undo any changes.
 
 
-### Switch to the Dart 2.12 release
+### Switch to the latest stable Dart release
 
 Switch to the **latest stable release**
 of either the Dart SDK or the Flutter SDK.
```

Comment:
Not necessary, particularly with this change, but do you think it would be worth it to link to get [Get Dart](https://dart.dev/get-dart) and [Upgrading the Flutter SDK](https://flutter.dev/docs/development/tools/sdk/upgrading) here?

Reply:
Yep, it probably would be.

Reply:
Wow, can't believe I let a typo slip through!

Reply:
Dart elease sounds like a dessert or an electronic lease!

Reply:
Or a misspelling of Élise!